packages/venia-ui/lib/components/CommunicationsPage/communicationsPage.css

```diff
@@ -0,0 +1,50 @@
+.root {
+    padding: 20px;
+    font-size: 0.875rem;
+}
+
+.header {
+    padding-bottom: 10px;
+}
+
+.title {
+    margin-bottom: 25px;
```

Comment:
Rarely should there be a need for odd values; we also have a strong preference for CSS Grid for layouts with gaps for spacing. If you change `.root` to something like this, it should flow just the same.

```css
.root {
    display: grid;
    row-gap: 1rem;
}
```